taxi/urls.py

```diff
@@ -16,11 +22,31 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>update/", CarUpdateView.as_view(), name="car-update"),
+    path("cars/<int:pk>delete/", CarDeleteView.as_view(), name="car-delete"),
```

Comment:
There is no slashes after pk. Is that expected? 
In this case your link will look like cars/1update/
When it should look like cars/1/update/